src/components/VPopover/VPopover.vue

```diff
@@ -104,6 +104,10 @@ export default defineComponent({
      * @default undefined
      */
     labelledBy: { type: String },
+    /**
+     * the z-index to apply to the popover content
+     */
+    zPos: { type: Number },
```

Comment:
This also needs to be passed to the `<VPopoverContent>` rendered in the template.

Could we rename it to `zIndex`, just for consistency with the actual name of the property being manipulated?

Reply:
Fixed in b1b9482.